test/server/reporter-test.js

```diff
@@ -830,7 +861,12 @@ describe('Reporter', () => {
                         quarantine:     null,
                         screenshotPath: null,
                         screenshots:    [],
-                        videos:         []
+                        videos:         [],
+                        testId:         'idf3t1',
+                        testRunIds:     [
+                            'f3t1',
+                            'f3t1'
```

Comment:
Please add different names to testRunIdMock if it's possible. It looks like I missed to this in previous PRs